docs/book/src/developers/development.md

```diff
@@ -112,6 +112,8 @@ Makefile targets and scripts are offered to work with go modules:
 - `make verify-modules` checks whether go module files are out of date.
 - `make modules` runs `go mod tidy` to ensure proper vendoring.
 - `hack/ensure-go.sh` checks that the Go version and environment variables are properly set.
+- `make lint` checks for the possible lint errors in the code base.
```

Comment:
this section is specifically about go modules, not sure the lint targets belong in here

Reply:
Removed it.